packages/hdwallet-provider/src/index.ts

```diff
@@ -154,7 +176,7 @@ class HDWalletProvider {
         },
         signPersonalMessage(...args: any[]) {
           this.signMessage(...args);
-        }
+        },
```

Comment:
These changes might be your prettier rules?

Reply:
Yep, it is from prettier.

Reply:
🤦